jerry-core/parser/js/js-parser-internal.h

```diff
@@ -273,6 +273,19 @@ typedef struct
 #endif /* PARSER_DUMP_BYTE_CODE */
 } parser_context_t;
 
+/**
+ * @}
+ * @}
+ * @}
+ */
+
+/** \addtogroup mem Memory allocation
+ * @{
+ *
+ * \addtogroup mem_parser Parser memory manager
+ * @{
+ */
+
```

Comment:
It's needed because the implementations of the functions defined in this header are put in these different groups. So either the header should group them separately just like the sources do, or the sources should put all the implementations in the same group as the header did.